### ISSUE TITLE
Build and debug Apollo 5.0 using VSCode

### ISSUE THREAD

**Issue by Avps1** [ApolloAuto/apollo#10618]
**Bug Description**
Clicking ''Run Build Task'' or pressing the keyboard shortcut (Ctrl+Shift+B) shows ''No build task to run found'' (vide second ScreenShot). 

**Steps to reproduce the behavior**
1. Go to 'https://github.com/ApolloAuto/apollo/blob/master/docs/howto/how_to_build_and_release.md'
2. Scroll down to 'Build the Apollo project in VSCode' .
3. Do all the steps under ''Build Apollo''

**Expected behavior**
Apollo 5.0 should be built, just like Apollo 3.5 was built(See first Screenshot).
The message ''NO build task to run found'' should not be displayed. It wasnt displayed for Apollo 3.5.

**Screenshots**
_Scrn1:_
![Apollo_3 5_Build](https://user-images.githubusercontent.com/50120641/72291489-c0039d00-364f-11ea-83d6-e113c4cb2d39.png)
_Scrn2:_
![Apollo_5 0_Build_Error](https://user-images.githubusercontent.com/50120641/72291499-c4c85100-364f-11ea-9de0-346714adf249.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04

**Additional context**
Please note that I have done the exact same steps for Apollo 3.5, like getting inside the docker first through terminal, then opening the right folder and building through VSCode, and everything was running exactly as stated in the page .

**Comment by aaronaaronson98** [comment]
I see that your image doesn't show a `.vscode/tasks.json` file. Try adding [this](https://github.com/ApolloAuto/apollo/blob/r5.0.0/.vscode/tasks.json) one?

**Comment by Avps1** [comment]
@aaronaaronson98  : Thank you so much for your answer. The problem is solved after I added tasks.json under .vscode. I am closing this issue now. But here's a suggestion: kindly consider if this qualifies as a bug, because I think tasks.json should be downloaded **_automatically_** after apollo-5.0 image is downloaded via the command `docker pull lgsvl/apollo-5.0`. I followed the steps mentioned in [(https://github.com/lgsvl/apollo-5.0)] to download apollo-5.0 in my machine. That was always the case with the previous Apollo versions .
